CONTRIBUTING.md

```diff
@@ -99,7 +99,7 @@ In case the test setup has trouble locating your `composer.phar` file:
     </php>
     ```
     **Note**: this setting also allows for locally testing with different versions of Composer.
-    You could, for instance, have multiple Composer Phar files locally, `composer1.phar`, `composer2.1.phar`, `composer2.2.phar`.
+    You could, for instance, have multiple Composer Phar files locally, `composer.phar`, `composer2.1.phar`, `composer2.2.phar`.
```

Comment:
```suggestion
    You could, for instance, have multiple Composer Phar files locally, `composer.phar`, `composer2.2.phar`, `composer2.6.phar`.
```

If we're dropping support for Composer < 2.2, mentioning a `2.1` phar is also confusing.

Reply:
Good catch. I'll update these version numbers too.